tests/src/Kernel/EntityBufferTest.php

```diff
@@ -10,6 +10,9 @@
  */
 class EntityBufferTest extends GraphQLTestBase {
 
+  /**
+   * @var NodeIDsArray
```

Comment:
Actually I found out that the test is written incorrectly. Problem is with the following code in the file below:
```
    foreach (range(1, 3) as $i) {
      $this->nodeIds[] = Node::create([
        'title' => 'Node ' . $i,
        'type' => 'test',
      ])->save();
    }
```
Because `save` method below does not return entity (node) id, it returns the status of saving the entity (for new entities it returns 1, for updated entities returns 2, for deleted entities it returns 3). So the array is filled with three 1's which is something we don't want. Let's first fix the `foreach` loop bellow with the following form:
```
    foreach (range(1, 3) as $i) {
      $node = Node::create([
        'title' => 'Node ' . $i,
        'type' => 'test',
      ]);
      $node->save();
      $this->nodeIds[] = $node->id();
    }
```

And now let's fix this `@var` annotation. `@var` is describing the type of the variable, so it can be `string`, `int`, `bool`, or some specific object like `\Drupal\Tests\graphql\Kernel\EntityBufferTest`. In this case it's array. For array we always want to be as specific as possible. If it's array of booleans then we'll use `bool[]`. Here we know that `$node->id()` returns id as a string, so let's use this:
```
   * @var string[]
```